build.gradle

```diff
@@ -337,6 +337,7 @@ dependencies {
         }
     }
     implementation group: "org.apache.commons", name: "commons-text", version: "1.9"
+    implementation group: "org.apache.avro", name: "avro", version: "1.8.2"
```

Comment:
This is used for preconditions, but isn't explicitly imported in tanagra. I imported it specifically here, but I don't even know if we need it (see comment in `ValidationUtils`